rero_ils/config.py

```diff
@@ -1980,3 +1980,29 @@ def _(x):
         import_size=50
     )
 )
+
+# SIP2
+# ====
+SIP2_SUPPORT_CHECKIN = True
```

Comment:
Is it `SIP2_ALLOW_CHECKIN`? or really `SIP2_SUPPORT_CHECKIN`

Reply:
In a system status response, we need to return `checkin_ok` and `supported_message` .

```
checkin_ok: indicates that the SC is allowed to check in items.
supported_messages: This field is used to notify the SC about which messages the ACS supports.
```

`SIP2_SUPPORT_CHECKIN` is used for `supported_message` field.

At this time `checkin_ok` in not really implemented in invenio-sip2 module.